contributor-docs/adrs/adr-005-token-prefix.md

```diff
@@ -0,0 +1,30 @@
+# Prefix for primitives
+
+## Status
+
+| Stage    | Status      |
+| -------- | ----------- |
+| Approved | 🚧 |
+| Adopted  | <!-- 🚧 --> |
+
+## Context
+
+Currently primitives have no prefix, which means once converted to css they can be used like `--fgColor-default`. This is very convenient but could lead to naming collisions or confusion if folks use their own css variables e.g. is `--button-fgColor-default` a token coming from primitive or a local variable?
+
+To solve this we could add a prefix like `primer` or `product` to our tokens, resulting in css variables like `--primer-fgColor-default`.
+
+## Decision
+
+The following decisions need ot be made:
+
+1. Do we think that this is a legit problem that is serious enough to invest somewhat significant resources into?
+2. If yes, what is an appropriate prefix?
+3. If no, do we need some kind of guidance in primer.style, or are we good without it?
```

Comment:
What would the guidance be? I don't think CSS variable collisions are specific enough to Primer to need documentation.

Reply:
> What would the guidance be? I don't think CSS variable collisions are specific enough to Primer to need documentation.

Idk. Maybe suggesting using a custom prefix if you create your own tokens? Or outlining all the token types we have so that people can avoid conflicts.

But as I mentioned above, I am not necessarily a _"fan"_ of adding a prefix (mainly because of the significant resource demand). And I also don't really have a point for 3. yet. Just wondering if there is any.